susemanager-utils/susemanager-sls/src/modules/sumautil.py

```diff
@@ -117,9 +117,12 @@ def get_net_module(iface):
     """
     sysfspath = os.path.join(SYSFS_NET_PATH, iface, "device/driver")
 
-    return (
-        os.path.exists(sysfspath) and os.path.split(os.readlink(sysfspath))[-1] or None
-    )
+    if os.path.exists(sysfspath):
+        return os.path.split(os.readlink(sysfspath))[-1] or None
+    elif os.path.exists(os.path.join(SYSFS_NET_PATH, iface, "bridge")):
```

Comment:
Can we give this a name? Above we use `sysfspath`, which is now ambiguous. For consistency it would be good to give both of them a name

Reply:
fixed